.vscode/settings.json

```diff
@@ -51,7 +51,7 @@
     "editor.defaultFormatter": "esbenp.prettier-vscode"
   },
   "[typescript]": {
-    "editor.defaultFormatter": "esbenp.prettier-vscode"
+    "editor.defaultFormatter": "vscode.typescript-language-features"
```

Comment:
Is one better? 😅

Reply:
We shouldn't be updating vscode settings for everyone without discussing it first, please revert changes to this file

Reply:
This change is not intentional, something in my VS code updated this. I will restore this.